example/95-audit-policy-for-shoot.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
All example file names start with the `kind`, hence, please rename this file to e.g. `95-configmap-custom-audit-policy.yaml`.